### ISSUE TITLE
How to train with new pictures?

### ISSUE THREAD

**Issue by PengPeng-JunJun** [ultralytics/yolov5#13357]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi, I have to train as code YOLOV5-6.1 and get good result.Now , I add some new images to the dataset.I want to know how to continue training on the best.pt before.
And please tell me how to call this operation,thanks!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @PengPeng-JunJun, thank you for your interest in YOLOv5 🚀!

Please visit our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5/) to get started with tasks like [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/) and more advanced concepts.

For your question about continuing training, you can load your `best.pt` model and resume training with your updated dataset. This process is often referred to as "fine-tuning" or "continued training."

If this relates to a 🐛 Bug Report, please provide a **minimum reproducible example** to help us debug it.

If you have any custom training ❓ questions, please include details such as dataset examples and logs, and ensure you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

## Requirements

Ensure you have [**Python>=3.8.0**](https://www.python.org/) and install all dependencies in [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt), including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):
```bash
git clone https://github.com/ultralytics/yolov5
cd yolov5
pip install -r requirements.txt
```

## Environments

YOLOv5 can be run in the following environments, which include all necessary dependencies:

- **Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon EC2**: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>

A green badge indicates all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) tests are passing.

## Introducing YOLOv8 🚀

We are thrilled to introduce [YOLOv8](https://github.com/ultralytics/ultralytics) 🚀! Ideal for object detection, image segmentation, and classification tasks. Check out the [YOLOv8 Docs](https://docs.ultralytics.com/) and get started:
```bash
pip install ultralytics
```

An Ultralytics engineer will follow up with you soon! 😊

**Comment by pderrenger** [comment]
@PengPeng-JunJun to continue training with new images using your existing `
